specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2021-03-01-preview/mfe.json

```diff
@@ -0,0 +1,4932 @@
+          }
+        },
+        "nextLink": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": false
+    },
+    "ModelVersion": {
+      "description": "Details of an ModelDto",
+      "type": "object",
+      "properties": {
+        "stage": {
+          "description": "Model asset stage.",
+          "type": "string"
+        },
+        "datastoreId": {
+          "description": "The asset datastoreId",
+          "type": "string"
+        },
+        "assetPath": {
+          "description": "Details of asset path in the datastore.",
+          "allOf": [
+            {
+              "$ref": "#/definitions/AssetPath"
+            }
+          ]
+        },
+        "description": {
+          "description": "The asset description text.",
+          "type": "string"
+        },
+        "tags": {
+          "description": "Tag dictionary. Tags can be added, removed, and updated.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "properties": {
+          "description": "The asset property dictionary.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "x-ms-client-flatten": true
+        }
+      },
+      "additionalProperties": false
+    },
+    "ModelVersionResource": {
+      "description": "Azure Resource Manager resource envelope.",
+      "required": [
+        "properties"
+      ],
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The resource URL of the entity (not URL encoded).",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "The name of the resource entity.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "The resource provider and type.",
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "description": "Additional attributes of the entity.",
+          "allOf": [
+            {
+              "$ref": "#/definitions/ModelVersion"
+            }
+          ],
+          "x-ms-client-flatten": true
+        },
+        "systemData": {
+          "description": "System data associated with resource provider",
+          "allOf": [
+            {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData"
+            }
+          ],
+          "readOnly": true
+        }
+      },
+      "x-ms-azure-resource": true,
+      "additionalProperties": false
+    },
+    "ModelContainer": {
+      "type": "object",
+      "properties": {
+        "latestVersions": {
+          "description": "Latest model versions for each stage.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ModelVersionResource"
```

Comment:
Do not duplicate child resources inside of the parent.  You can list resource IDs here if necessary to indicate which is the latest but the content of the children must be retrieved by doing a GET collection or single GET call to ensure RBAC is enforced